libraries/Microsoft.Bot.Schema/Teams/Generated/MessageActionsPayload.cs

```diff
@@ -1,4 +1,4 @@
-// <auto-generated>
+﻿// <auto-generated>
```

Comment:
This line can be deleted, as this was initially auto-generated, but from here on out is hand-coded.